lib/execution_engine2/utils/Condor.py

```diff
@@ -353,7 +353,7 @@ def cancel_job(self, job_id: str) -> bool:
         :param job_id:
         :return:
         """
-        return self.cancel_jobs([job_id])
+        return self.cancel_jobs(["job_id"])
```

Comment:
I think this line shouldn't be changed.

Reply:
@Tianhao-Gu Fixed it thanks. Condor doesn't like` ints`